RecoLocalTracker/SiPixelRecHits/plugins/PixelCPEGenericESProducer.cc

```diff
@@ -42,23 +43,20 @@ using namespace edm;
 PixelCPEGenericESProducer::PixelCPEGenericESProducer(const edm::ParameterSet& p) {
   std::string myname = p.getParameter<std::string>("ComponentName");
   // Use LA-width from DB. If both (upper and this) are false LA-width is calcuated from LA-offset
-  useLAWidthFromDB_ = p.existsAs<bool>("useLAWidthFromDB") ? p.getParameter<bool>("useLAWidthFromDB") : false;
+  useLAWidthFromDB_ = p.getParameter<bool>("useLAWidthFromDB");
   // Use Alignment LA-offset
-  const bool useLAAlignmentOffsets =
-      p.existsAs<bool>("useLAAlignmentOffsets") ? p.getParameter<bool>("useLAAlignmentOffsets") : false;
+  const bool useLAAlignmentOffsets = p.getParameter<bool>("useLAAlignmentOffsets");
   char const* laLabel = "";  // standard LA, from calibration, label=""
   if (useLAAlignmentOffsets) {
     laLabel = "fromAlignment";
   }
 
-  auto magname = p.existsAs<edm::ESInputTag>("MagneticFieldRecord")
-                     ? p.getParameter<edm::ESInputTag>("MagneticFieldRecord")
-                     : edm::ESInputTag("");
+  auto magname = p.getParameter<edm::ESInputTag>("MagneticFieldRecord");
   UseErrorsFromTemplates_ = p.getParameter<bool>("UseErrorsFromTemplates");
 
   pset_ = p;
   auto c = setWhatProduced(this, myname);
-  c.setConsumes(magfieldToken_, magname)
+  c.setConsumes(magfieldToken_, edm::ESInputTag(magname))
```

Comment:
Why do a copy constructor as `magname` is already an `ESInputTag`?

Reply:
@Dr15Jones, sorry it was a leftover from an earlier iteration. Fixed now.